pkg/universe.dagger.io/go/test/test.cue

```diff
@@ -8,8 +8,20 @@ import (
 dagger.#Plan & {
 	client: filesystem: "./data/hello": read: contents: dagger.#FS
 
-	actions: test: go.#Test & {
-		source:  client.filesystem."./data/hello".read.contents
-		package: "./greeting"
+	actions: test: {
+		simple: go.#Test & {
+			source: client.filesystem."./data/hello".read.contents
+		}
+
+		withPackage: go.#Test & {
+			source:  client.filesystem."./data/hello".read.contents
+			package: "./greeting"
+		}
+
+		withPackages: go.#Test & {
+			source: client.filesystem."./data/hello".read.contents
+			packages: ["./greeting", "./math"]
+		}
+
```

Comment:
Mixing mix test

Reply:
I don't understand your comment :stuck_out_tongue:

Reply:
I think I got it: `missing a test mixing package + packages`
You're right and I'll add this.

Reply:
Yes haha sorry

Reply:
Added it.
I tried to find a better way to do it, as I create temporary files for the right execution of each package under test.
Before each execution, my tests cleans up the FS.
I'd rather have it working without mount, but I don't know how to do that. I'll think it will be **good enough** for now.

Reply:
That currently do not work on my localhost, just executiong `go.#Test` should be enough no?